flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -341,6 +347,18 @@ under the License.
 									<pattern>com.fasterxml</pattern>
 									<shadedPattern>org.apache.flink.calcite.shaded.com.fasterxml</shadedPattern>
 								</relocation>
+								<relocation>
```

Comment:
Why do we need Apache commons? Actually it took me quite some time last year to reduce the number of potentially conflicting dependencies. `EncodingUtils` has the most commonly used methods. Just adding a dependency because of one method doesn't seem right.

Reply:
It's introduced by calcite 1.20: `org.apache.calcite.runtime.SqlFunctions` used `org.apache.commons.codec.language.Soundex`